inferno/lib/notifications.py

```diff
@@ -27,3 +30,29 @@ def send_mail(job_id=None, job_fail=None, mail_to=None, mail_from=None, mail_ser
         return True
     except:
         return False
+
+
+def send_pagerduty(job_id=None, job_fail=None,
+                   api_key=None, retry=None, retry_delay=None):
+    api_url = "https://events.pagerduty.com/generic/2010-04-15/create_event.json"
+    if not job_id or not job_fail:
+        raise Exception(
+            "Missing job_id or job_fail for Pagerduty notification")
+    if not api_key:
+        raise Exception(
+            "Missing pagerduty API key")
+    if retry and retry_delay:
```

Comment:
This retry stuff is un-used as part of this send_pagerduty method.  Should be removed or appended to the pd_data['details'] perhaps!